disnake/player.py

```diff
@@ -274,6 +281,7 @@ def __init__(
         super().__init__(source, executable=executable, args=args, **subprocess_kwargs)
 
     def read(self) -> bytes:
+        FFmpegAudio.read(self)
```

Comment:
I really don't like the way you call this, why not a normal method call?

Reply:
So, we define the default reading method for heirs and do not repeat the code that can be added to the base "read" method.

Reply:
That'd be all well and good but currently, it doesn't do any reading does it? I'm all for moving common code to another function later but this is not a place where that's needed.

PS: You swapped your messages around

Reply:
As for me, this is a good place, if we need more checks, then we won't need to write the same code in 2 'read' functions, or even more if we make more heirs

Reply:
You misunderstood me, I was just not to keen on using `read()` for a function which performs a check, rather a name like `check_if_read()` would be better (although I'm not that great at naming things myself so you might come up with an even better one)

Reply:
In general, your logic is also logical, but I called the method 'read` because usually methods are not replaced during inheritance, but supplemented, and I wanted to reflect this in the code so that this method is used in `read' during inheritance, it is not necessary that there will be only this check

But I like your version, it's also logical, you can use the prefix 'check` and also about what the function does, for example, I would call: 'check_process` or in a similar way, I'm not good at names

Sorry for the English lang, this is partly a translator:3

Reply:
What about `check_streams()` for the name?

Reply:
It seems not bad, but it is unclear what kind of 'stream`

Reply:
It's plural so all streams

Reply:
I still haven't figured out which `streams`

Reply:
stdin, stdout and stderr are called "streams"

Reply:
Something more global is needed here than just 'streams`, because it's not about streams, but that its are 'MISSING' only when 'FFmpegAudio` is cleaned and cannot be used
hm

Reply:
Does it even need to be a function of the base class though?

Reply:
I think yes, because it is not used in one heir

Reply:
Alright, do that then. 👍

Reply:
What exactly? I didn't understand:3
You're the one who wanted to change the code, I don't mind